SPECS-EXTENDED/rubygem-abrt/rubygem-abrt.spec

```diff
@@ -1,29 +1,24 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-# Generated from abrt-0.0.2.gem by gem2rpm -*- rpm-spec -*-
 %global gem_name abrt
-
-Name: rubygem-%{gem_name}
-Version: 0.4.0
-Release: 2%{?dist}
-Summary: ABRT support for Ruby
-License: MIT
-URL: http://github.com/voxik/abrt-ruby
-Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
-# git clone https://github.com/voxik/abrt-ruby.git && cd abrt-ruby
-# git archive -v -o abrt-0.4.0-spec.tar.gz v0.4.0 spec/
-Source1: %{gem_name}-%{version}-spec.tar.gz
-Requires: libreport-filesystem
+Summary:       ABRT support for Ruby
+Name:          rubygem-%{gem_name}
+Version:       0.4.0
+Release:       2%{?dist}
```

Comment:
Please update the `Release` number and changelog entry. This is a substantial change from what we had.